src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -199,6 +199,17 @@ public DynamoView(DynamoViewModel dynamoViewModel)
             FocusableGrid.InputBindings.Clear();
         }
 
+        /// <summary>
+        /// This method close a tab item in the right side bar based on passed extension
+        /// </summary>
+        /// <param name="viewExtension">Extension to be closed</param>
+        /// <returns></returns>
+        internal void CloseTabItem(IViewExtension viewExtension)
+        {
+            string tabName = viewExtension.Name;
```

Comment:
uuid might be safer?

Reply:
@reddyashish @mjkkirschner Will address this in a new task